spec.bs

```diff
@@ -1314,8 +1341,28 @@ Several APIs specific to fenced frames are defined on the {{Fence}} interface.
         : [=fencedframeconfig/urn=]
         :: |urn|
 
-        : [=fencedframeconfig/config=]
-        :: |config|
+        : [=fencedframeconfig/sharedStorageContext=]
+        :: |config|'s [=fenced frame config/embedder shared storage context=]
+
+        : [=fencedframeconfig/containerWidth=]
+        :: null if |config|'s [=fenced frame config/container size=] is null, otherwise |config|'s
+           [=fenced frame config/container size=]'s [=size/width=]
+
+        : [=fencedframeconfig/containerHeight=]
+        :: null if |config|'s [=fenced frame config/container size=] is null, otherwise |config|'s
+           [=fenced frame config/container size=]'s [=size/height=]
+
+        : [=fencedframeconfig/contentWidth=]
+        :: null if |config|'s [=fenced frame config/content size=] is null, {{OpaqueProperty}} if
```

Comment:
We cant just use `{{OpaqueProperty}}`. See the other usages of this where we have to specify a value (which happens to be the only single value available).